linux_os/guide/system/accounts/accounts-pam/display_login_attempts/rule.yml

```diff
@@ -4,11 +4,18 @@ title: 'Ensure PAM Displays Last Logon/Access Notification'
 
 description: |-
     To configure the system to notify users of last logon/access
-    using <tt>pam_lastlog</tt>, add or correct the <tt>pam_lastlog</tt> settings in
+    using <tt>pam_lastlog</tt>, add or correct the <tt>pam_lastlog</tt>
+    settings in
+    {{% if product == "sle12" %}}
+    <tt>/etc/pam.d/login</tt> to read as follows:
+    <pre>session     required pam_lastlog.so showfailed</pre>
+    And make sure that, the <tt>silent</tt> option is not set.
```

Comment:
```suggestion
    And make sure that the <tt>silent</tt> option is not set.
```

Reply:
Thanks, I will get this addressed and update the PR.

Reply:
done